task_12/src/_variables.scss

```diff
@@ -0,0 +1,10 @@
+$font : 'Roboto';
+
+$green: #4F7513;
```

Comment:
Pretty much good.

Commonly people use something like `$color-primary`, `$color-secondary`, `$color-warning` and so on. You may see it in the frameworks like Bootstrap. It's does not matter but that way allow you to change color scheme easily without changing meaning of variables.